src/test/groovy/graphql/schema/GraphQLArgumentTest.groovy

```diff
@@ -195,23 +195,53 @@ class GraphQLArgumentTest extends Specification {
         resolvedDefaultValue == null
     }
 
-    def "Applied schema directives arguments are validated for programmatic schemas"() {
```

Comment:
Yeah the reason this is confusing is explained on the main PR comments.  Nominally a type should have BOTH a GraphQLAppliedDirective for every GraphQLDirective - but tests allow this not to happen - or more properly programatic building of schemas allow it (since it could also happen in production code if you build your schema directly and not via `graphql.schema.idl.SchemaGenerator`).

Reply:
See `graphql.schema.idl.SchemaGeneratorAppliedDirectiveHelper`